internal/datanode/event_manager.go

```diff
@@ -291,7 +291,7 @@ func isEndWatchState(state datapb.ChannelWatchState) bool {
 		state != datapb.ChannelWatchState_Uncomplete // legacy state, equal to ToWatch
 }
 
-type tickler struct {
+type etcdTickler struct {
```

Comment:
tickler is better than etcdTickler

Reply:
Will add a new tickler without etcd operation later, etcdTickler will be deprecated soon, so I changed the name.

Reply:
ok